app/views/pages/help.html.slim

```diff
@@ -0,0 +1 @@
+= render partial: 'goals_of_triage'
```

Comment:
`render 'goals_of_triage'` should work!

Reply:
Agreed - I was trying to keep my edits to the original PR minimal until it's been reviewed.  @nateberkopec is this something you want to see developed further or should we close this and the original PR?